README.md

```diff
@@ -14,7 +14,7 @@ I do not support piracy in any way, all the data and information in my software
 ## Current functionality
 ### General
 - Support for Web Browser (SPIDER) entrypoint for Old3DS (ver 4.1-9.2)
-- Support for System Settings (MSET) entrypoint for Old3DS (ver 4.1-4.5 and 6.0-6.4)
+- Support for System Settings (MSET) entrypoint for Old3DS (6.0-6.4)
```

Comment:
Unless I'm misunderstanding something, the issue I submitted last night (
https://github.com/roxas75/rxTools/issues/245#issuecomment-139990259) was
solved by using 6.x MSET instead of 4.x. I could be doing something wrong,
but on my O3DS it wouldn't work with 4.x MSET.
On Mon, Sep 14, 2015 at 12:29 PM nastys <notifications@github.com> wrote:

> In README.md
> <https://github.com/roxas75/rxTools/pull/246#discussion_r39414914>:
>
> > @@ -14,7 +14,7 @@ I do not support piracy in any way, all the data and information in my software
> >  ## Current functionality
> >  ### General
> >  - Support for Web Browser (SPIDER) entrypoint for Old3DS (ver 4.1-9.2)
> > -- Support for System Settings (MSET) entrypoint for Old3DS (ver 4.1-4.5 and 6.0-6.4)
> > +- Support for System Settings (MSET) entrypoint for Old3DS (6.0-6.4)
>
> Why did you remove 4.1-4.5?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/roxas75/rxTools/pull/246/files#r39414914>.
>